src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/partials/SidebarBanner.tsx

```diff
@@ -88,6 +100,13 @@ export const SidebarBanner: FC = () => {
           </NotificationBanner>
         </div>
       ))}
+      {showVersionUpToDateNotification && (
+        <div className="mt-6">
+          <NotificationBanner icon="check-circle" status="success">
```

Comment:
Why is the status of `NotificationBanner` a plain string and not an `enum`.

_(This is not directed at you @iamsamgibbs, it's just me ranting about the implemetation)_

Reply:
Opened as new issue: #1770